attribute/multivalueattribute.go

```diff
@@ -29,40 +33,55 @@ func NewMultiValue(a *yotiprotoattr.Attribute) (*MultiValueAttribute, error) {
 	}, nil
 }
 
-// ParseMultiValue recursively unmarshals and converts Multi Value bytes into a slice of Items
-func ParseMultiValue(data []byte) []*Item {
+// parseMultiValue recursively unmarshals and converts Multi Value bytes into a slice of Items
+func parseMultiValue(data []byte) ([]*Item, error) {
```

Comment:
(This hasn't been released yet, so changing it from exported to unexported is not a breaking change)

Reply:
Sure.  But why would SDK callers use this method?  Don't we parse everything up front, then just give them the result?

Reply:
Yes, this was a remnant of the the first PR on MultiValue when we were giving clients the option to parse multi values. Now we should handle everything, so that won't be needed, so is not exported.